src/redux/appReduxer.js

```diff
@@ -0,0 +1,127 @@
+import {getFormate} from "../utils.js";
+
+const ADD_SESSION_LENGTH = 'ADD_SESSION_LENGTH'
+const ADD_BREAK_LENGTH = 'ADD_BREAK_LENGTH'
```

Comment:
The word increase/decrease fits better in this case. It sounds good in Russian, but in English it sounds a bit strange

Reply:
`
const CHANGE_SESSION_LENGTH = 'CHANGE_SESSION_LENGTH'
const CHANGE_BREAK_LENGTH = 'CHANGE_BREAK_LENGTH'
`
So?